vendor/assets/leaflet/leaflet.osm.js

```diff
@@ -1,8 +1,25 @@
 L.OSM = {};
 
+var h2 = false;
+// tile.openstreetmap.org supports http/2 where sharding makes no sense.
+// Use only one subdomain if we're using http/2
+if (
+    (window.performance && window.performance.getEntries      && performance.getEntries()[0].nextHopProtocol == 'h2') ||
+    (window.performance && performance.timing.nextHopProtocol && performance.timing.nextHopProtocol          == 'h2') ||
+    (window.chrome      && window.chrome.loadTimes            && window.chrome.loadTimes().connectionInfo    == 'h2')
+) {
+    h2 = true;
+}
+
+if (h2) {
+    var osmtileurl = 'https://a.tile.openstreetmap.org/{z}/{x}/{y}.png';
```

Comment:
I recommend using `tile.openstreetmap.org` if you don't need the abc subdomains, since the `a` is just an artifact of the load distribution that this patch moves away from.